app/selectors/favorites.js

```diff
@@ -15,3 +15,26 @@ export function getFavoriteTrack(state, artist, title) {
     return normalizedStoreArtist === normalizedArtist && normalizedStoreTitle === normalizedTitle;
   });
 }
+
+export function getFavoriteAlbum(state, artist, title) {
+  if (!artist || !title) {
+    return null;
+  }
+
+  const normalizedArtist = _.deburr(artist.toLowerCase());
+  const normalizedTitle = _.deburr(title.toLowerCase());
+
+  return _.find(state.favorites.albums, album => {
+    const albumArtist = album.artists[0];
+
+    if (!albumArtist) {
+      return false;
+    }
+
+    const artistQuantifierRegex = /\s\([0-9]*\)$/;
```

Comment:
Regex that removes the last occurrence of `({number})` in artist name like `Florist (2)`. It should ignore any parentheses inside. May fail in case of any artist really ending with `({number)`

Reply:
Yep, I didn't realize that. It looks like this is an internal Discogs suffix added to distinguish between artists with the same name. Doesn't help us since we look at ids anyway.